webapp/src/Service/ExternalContestSourceService.php

```diff
@@ -766,7 +766,7 @@ protected function validateAndUpdateContest(Event $event, EventData $data): void
 
         // Also compare the penalty time
         $penaltyTime = $data->penaltyTime;
-        if ($this->config->get('penalty_time') != $penaltyTime) {
+        if ($penaltyTime !== null & $this->config->get('penalty_time') != $penaltyTime) {
```

Comment:
Bitwise-and becomes a truthy value and is the same in this case, but does not seem intentional.